spec/lib/fidor_api/model/scheduled_transfer_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+module FidorApi
+  module Model
+    RSpec.describe ScheduledTransfer do
```

Comment:
Can we please get rid of the complete file? As mentioned before we don't need to test the attributes mechanism since it's provided by another library.